foyer/forcefields/xml/oplsaa.xml

```diff
@@ -259,11 +259,11 @@
   <Type name="opls_257" class="opls_257" element="C" mass="12.011"/>
   <Type name="opls_258" class="opls_258" element="C" mass="12.011"/>
   <Type name="opls_259" class="opls_259" element="C" mass="12.011"/>
-  <Type name="opls_260" class="opls_260" element="C" mass="12.011"/>
-  <Type name="opls_261" class="opls_261" element="C" mass="12.011"/>
-  <Type name="opls_262" class="opls_262" element="N" mass="14.0067"/>
-  <Type name="opls_263" class="opls_263" element="C" mass="12.011"/>
-  <Type name="opls_264" class="opls_264" element="Cl" mass="35.453"/>
+  <Type name="opls_260" class="CA" element="C" mass="12.011" def="[C;X3;r6]([C;X2;%opls_261])" overrides = "opls_145" desc="Benzonitrile C(CN)" doi="10.1021/ja9621760"/>
+  <Type name="opls_261" class="CZ" element="C" mass="12.011" def="[C;X2]([C;X3;r6])" overrides = "opls_754" desc="Benzonitrile (benzene)C(N)" doi="10.1021/ja9621760"/>
```

Comment:
The descriptions seem a bit confusing to me. Could they be re-worded a bit? For example:
```suggestion
  <Type name="opls_261" class="CZ" element="C" mass="12.011" def="[C;X2]([C;X3;r6])" overrides = "opls_754" desc="Benzonitrile: carbon in nitrile group (CN) " doi="10.1021/ja9621760"/>
```